packages/components/src/components/editor/condition-builder/ConditionBuilderContext.tsx

```diff
@@ -0,0 +1,155 @@
+import { createContext, useContext, useState, type ReactNode } from 'react';
+import type { Condition } from './Condition';
+import type { ConditionGroup } from './ConditionGroup';
+import type { ConditionMode } from './ConditionBuilder';
+import { Input } from '@/components/common/input/input';
+
+export type TypeOperator = {
+  'equal to': string;
+  'not equal to': string;
+  'is true': string;
+  'is false': string;
+  'is empty': string;
+  'is not empty': string;
+  'less than': string;
+  'greater than': string;
+  'less or equal to': string;
+  'greater or equal to': string;
+};
+
+export type ConditionType = keyof TypeOperator;
+
+export type LogicOperator = {
+  'and': string;
+  'or': string;
+};
+
+export type LogicalOperator = keyof LogicOperator;
```

Comment:
use one of them but not both :D
I would streamline this name with the "normal" operator type

Reply:
I renamed the types no like this, what do you think?
![grafik](https://github.com/user-attachments/assets/4a832002-0541-4708-89f6-e765919039d0)